lib/lighthouse_bgs/base.rb

```diff
@@ -141,7 +141,7 @@ def client
         convert_request_keys_to: :none,
         ssl_verify_mode: @ssl_verify_mode.to_sym
       }
-      options[:host] = @forward_proxy_url unless @forward_proxy_url.nil?
+      options[:host] = @forward_proxy_url.gsub(%r{^http://|^https://}, '') unless @forward_proxy_url.nil?
```

Comment:
`%r{^http://|^https://}`
could be written more succinctly as
`%r{^https?://}`

Reply:
yes, good catch, just updated